main.go

```diff
@@ -20,16 +20,24 @@ func main() {
 	}
 
 	cmd.Flags().StringVar(&config.ControllerName, "controller-name", "helm-controller", "Unique name to identify this controller that is added to all HelmCharts tracked by this controller. May be set via CONTROLLER_NAME env var.")
+	cli.SetFlagtoEnv(cmd, "controller-name", "CONTROLLER_NAME")
 	cmd.Flags().BoolVar(&config.Debug, "debug", false, "Turn on debug logging")
```

Comment:
This is pretty ugly, is this repetition of `cmd.Flags().XVar()` and `cli.SetFlagtoEnv()` really the best way to do this? Why not just switch to urfave since that's what we use for the rest of our projects?

Reply:
Yeah cobra doesn't support have ENV support built into its system, it relies on pflag or viper for that.

Reply:
wrangler-cli was also hiding this https://github.com/rancher/wrangler-cli/blob/master/builder.go#L136-L149

Reply:
It also splits comma-separated values into slices for you, nice...
https://github.com/rancher/wrangler-cli/blob/master/builder.go#L118-L126

Reply:
Yeah, but none of the flags in this repo had that tag set so it wasn't needed functionality.